va/va.go

```diff
@@ -642,8 +642,8 @@ func (va *ValidationAuthorityImpl) validateChallengeAndCAA(ctx context.Context,
 }
 
 func (va *ValidationAuthorityImpl) validateChallenge(ctx context.Context, identifier core.AcmeIdentifier, challenge core.Challenge) ([]core.ValidationRecord, *probs.ProblemDetails) {
-	if !challenge.IsSaneForValidation() {
-		return nil, probs.Malformed("Challenge failed sanity check.")
+	if err := challenge.CheckConsistencyForValidation(); err != nil {
+		return nil, probs.Malformed("Challenge failed sanity check. %v", err)
```

Comment:
Generally we would use "%s" here. Also, this should now say "consistency check:"

`"Challenge failed consistency check: %s", err`